src/components/config-core.tsx

```
@@ -8,6 +8,7 @@ import { useTranslation } from 'react-i18next'
 import { If } from 'react-if-lite'
 import { useCoreBreakingConfirm } from '@/hooks/use-core-breaking-confirm'
 import { store } from '@/store'
+import { silence } from '@/utils/silence'
 import { compareVersions } from '@/utils/core-version'
```

ESLint

[error] Expected "`@/utils/core-version`" to come before "`@/utils/silence`".

(perfectionist/sort-imports)